starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/ItemObject.java

```diff
@@ -0,0 +1,20 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by student on 10/25/16.
+ */
+public class ItemObject {
+    private String mItem;
```

Comment:
It might be more clear if you named this variable `mItemName` rather than just `mItem` - try to be as descriptive as possible when naming things to avoid confusion